src/geovista/geodesic.py

```diff
@@ -106,7 +106,7 @@
 """Lookup table for cubed-sphere panel name by panel index."""
 
 CSC: float = np.rad2deg(np.arcsin(1 / np.sqrt(3)))
-"""Latitude (degrees) of a cubed-sphere panel corner."""
+"""Absolute latitude (degrees) of all cubed-sphere panel corners."""
```

Comment:
```suggestion
"""Absolute latitude (degrees) of a cubed-sphere panel corner."""
```

Reply:
I understand what this object is, but only by reading the code base. Without this context its presence in the documentation is quite distracting/confusing. This is my poor attempt at making that less so, but it's not great. Could we just remove it from the documentation?

Reply:
abf85e24e2fc6171e60c0ba09e2915bbd58d6b9d